editor.planx.uk/src/ui/editor/SettingsSection.tsx

```diff
@@ -3,14 +3,9 @@ import { styled } from "@mui/material/styles";
 import { contentFlowSpacing } from "@planx/components/shared/Preview/Card";
 import React, { ReactNode } from "react";
 
-interface RootProps extends BoxProps {
-  background?: boolean;
-  "data-testid"?: string;
-}
-
 const Root = styled(Box, {
   shouldForwardProp: (prop) => prop !== "background",
-})<RootProps>(({ background, theme }) => ({
+})<BoxProps & { background?: boolean }>(({ background, theme }) => ({
```

Comment:
Had to revert `bgcolor` since `background` was a boolean to conditionally style the `ServiceSettings` component

Reply:
I found this when running tests and seeing the other places this component was used